rotkehlchen/assets/asset.py

```diff
@@ -566,6 +568,13 @@ def to_dict(self) -> dict[str, Any]:
     def get_decimals(self) -> int:
         return 18 if self.decimals is None else self.decimals
 
+    def is_liability(self) -> bool:
+        """Returns True if the token is a liability token, False if it's an asset token"""
+        return (
+            self.protocol == CPT_AAVE_V3 and
```

Comment:
what about Aave v2? Or other protocols?

Reply:
looks like Aave v1/v2 and other protocols are handled well with other logic, so I didn't add extra checks here..